symfony/templates/partials/workqueue-consents-advance-filters.html.twig

```diff
@@ -0,0 +1,135 @@
+<div class="panel-group" id="accordion">
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a class="accordion-toggle collapsed" data-toggle="collapse" href="#columns_group" id="columns_group_label">
+                    <i class="fa fa-columns" aria-hidden="true"></i> Columns
+                </a>
+                <small><a id="columns_reset" data-group-id="columns_group">Reset</a></small>
+            </h4>
+        </div>
+        <div id="columns_group" class="panel-collapse collapse">
+            <div class="panel-body">
+                <ul class="list-group">
+                    {% set columNumber = 5 %}
+                    {% for name, columnDef in columnsDef %}
+                        {% if columnDef.toggleColumn %}
+                            {% if columnDef.displayNames is defined %}
+                                {% for subFieldName, subFieldLabel in columnDef.displayNames %}
+                                    {% set checked = "column"~subFieldName in app.session.get('workQueueConsentColumns') ? 'checked' : '' %}
+                                    <li class="list-group-item checkbox">
+                                        <labeL><input type="checkbox" name="column{{ subFieldName }}" {{ checked }} class="toggle-vis" data-column="{{ columNumber }}"/> {{ subFieldLabel }}</label>
+                                    </li>
+                                    {% set columNumber = columNumber + 1 %}
+                                {% endfor %}
+                            {% else %}
+                                {% set checked = "column"~name in app.session.get('workQueueConsentColumns') ? 'checked' : '' %}
+                                <li class="list-group-item checkbox">
+                                    <label><input type="checkbox" name="column{{ name }}" {{ checked }} class="toggle-vis" data-column="{{ columNumber }}"/> {{ columnDef.displayName }}</label>
+                                </li>
+                                {% set columNumber = columNumber + 1 %}
+                            {% endif %}
+                        {% endif %}
+                    {% endfor %}
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h4 class="panel-title">
+                <a class="accordion-toggle collapsed" data-toggle="collapse" href="#participant_lookup_group" id="participant_lookup_group_label">
+                    <i class="fa fa-search" aria-hidden="true"></i> Participant Lookup
+                </a>
+                <small><a id="participant_lookup_reset" data-group-id="participant_lookup_group">Reset</a></small>
+            </h4>
+        </div>
+        <div id="participant_lookup_group" class="panel-collapse collapse">
```

Comment:
If you are in one of these text box fields and then hit enter, the export dialog is triggered. This is likely because this form isn't an actual form, and the export button has the tab focus. We don't have to have this filter submit on enter, but can you figure out a way to at least prevent the modal from unexpectedly triggering?